core/src/test/java/org/hibernate/ogm/backendtck/storedprocedures/PositionalParametersStoredProcedureCallTest.java

```diff
@@ -46,7 +44,7 @@
  * @author Sergey Chernolyas &amp;sergey_chernolyas@gmail.com&amp;
  */
 @SkipByGridDialect(
-		value = { HASHMAP, INFINISPAN, INFINISPAN_REMOTE, NEO4J_EMBEDDED, NEO4J_REMOTE },
+		value = { HASHMAP, INFINISPAN, INFINISPAN_REMOTE },
```

Comment:
This won't work on CI because the build doesn't load the stored procedures for the remote dialects